packages/media/src/seekVideo.js

```diff
@@ -27,6 +27,7 @@ function seekVideo(video, offset = 0.99) {
   }
 
   return new Promise((resolve, reject) => {
+    video.addEventListener('suspend', reject);
```

Comment:
Should we have this everywhere we adding video is a promise?

Reply:
In my testing earlier I didn‘t see the need for it. But I can do more testing next week.

Reply:
I think we do this in a follow up PR.